Apsitvarkom.Models/DTO/PollutedLocationDTO.cs

```diff
@@ -45,4 +46,24 @@ public PollutedLocationDTOValidator()
         // Validate all Location fields using its validator if it has a value
         RuleFor(dto => dto.Location!.Value).SetValidator(new LocationDTOValidator()).When(dto => dto.Location.HasValue);
     }
+
+    /// <summary>
+    /// Checks whether the input Id is parseable as <see cref="Guid"/>.
+    /// </summary>
+    /// <param name="id">String to be parsed.</param>
+    /// <returns>Flag of whether the validation was successful.</returns>
+    private bool BeValidGuid(string? id)
+    {
+        return Guid.TryParse(id, out _);
+    }
+
+    /// <summary>
+    /// Checks whether input string is parseable as Date.
```

Comment:
In docs of the last method you used `<see cref="Guid"/>`. You could have done the same here with `DateTime`